Sources/ShortcutFoundation/Core/Logger/Loggable.swift

```diff
@@ -0,0 +1,145 @@
+//
+// Loggable.swift
+// VSFoundation
+//
+// Created by Gabriel Sabadin on 2021-12-13.
+// Copyright Virtual Stores - 2021
```

Comment:
These dam headers sometimes I feel that our standard should just be to not have them at all.

Reply:
My bad, I was sloppy here. Thanks for checking it out